shared/chat/inbox/row/container.js

```diff
@@ -117,15 +120,31 @@ const makeSelector = conversationIDKey => {
   }
 }
 
-// $FlowIssue
-const ConnectedRow = pausableConnect(
-  (state: TypedState, {conversationIDKey}) => {
+const mapStateToProps = (state: TypedState, {conversationIDKey, teamname, channelname}) => {
```

Comment:
pull this out as the helper and make onSelectConversation not need a param anymore in the components